tests/UKHO.ERPFacade.API.FunctionalTests/Model/RoSJsonPayloadHelper.cs

```diff
@@ -0,0 +1,57 @@
+﻿
```

Comment:
Need to remove this file as currently not used.

Reply:
Removed the file